### DISCUSSION TITLE
What do you think about my Autopublish extension?

### DISCUSSION THREAD

**Discussion by GiovanniSalmeri** [datenstrom/community#745] [Ask a question] [answered]
Hello everybody, I made a new little [Autopublish extension](https://github.com/GiovanniSalmeri/yellow-autopublish). It gives the status "Draft" to any page with a date in the future. When the date arrives, the page is automatically published. Any comments are welcome. Thank you!

**Comment by markseuffert** [comment, marked as answer]
I reviewed this extension quite a while ago. Perhaps it could be made publicly available and no longer be experimental?

I kindly suggest to add a new screenshot. The current one shows not what it does, how about an advent calendar? 😀

**Reply by GiovanniSalmeri** [reply]
I'd be delighted to! It's also a way for me to resume my activity on Yellow after several months absorbed by work 😢. Thank you for the suggestion: indeed, the current screenshot is rather unremarkable 😁

**Comment by adegans** [comment]
This doesn’t store the draft status when the extension is removed. Right? It just assumes the meta data on pageload without actually saving any data?
